guides/common/modules/proc_synchronizing-templates-with-an-existing-repository.adoc

```diff
@@ -4,13 +4,33 @@
 Use this procedure to synchronize templates between your {ProjectServer} and an existing repository.
 
 .Procedure
-. If you want to use HTTPS to connect to the repository and you use a self-signed certificate authentication on your Git server, validate the certificate:
+. If you want to use HTTPS to connect to the repository and you use a self-signed certificate authority (CA) on your Git server:
+.. Create a new directory under the `/usr/share/foreman/` directory to store the Git configuration for the certificate:
 +
 [options="nowrap" subs="+quotes,verbatim,attributes"]
 ----
-# sudo -u foreman git config --global http.sslCAPath _Path_To_My_Certificate_
+# mkdir -p /usr/share/foreman/.config/git
 ----
-. If you want to use SSH to connect to the repository, perform the following steps:
+.. Create a file named `config` in the new directory:
++
+[options="nowrap" subs="+quotes,verbatim,attributes"]
+----
+# touch /usr/share/foreman/.config/git/config
```

Comment:
In my own home directory I see it has mode 0600 and I think in some cases git is unhappy with a world readable file, but I'm not entirely sure if you also need to chmod.

Reply:
0600 does sound like a good idea and looking at [16.2. Using repository sources](https://theforeman-foreman-documentation-preview-pr-3243.surge.sh/nightly/Administering_Project/index-satellite.html#using-repository-sources_admin) as a whole, I don't think it breaks anything further down the line..? I feel like we're already asking users to jump through too many hoops here but on the other hand, this could be a security concern.

@adamruzicka Would you be okay with my adding `# chmod 0600 /usr/share/foreman/.config/git/config`?

Reply:
Yes, fine by me

Reply:
> 0600 does sound like a good idea and looking at [16.2. Using repository sources](https://theforeman-foreman-documentation-preview-pr-3243.surge.sh/nightly/Administering_Project/index-satellite.html#using-repository-sources_admin) as a whole, I don't think it breaks anything further down the line..? I feel like we're already asking users to jump through too many hoops here but on the other hand, this could be a security concern.

The reason I ask is that I vague recall that git ignores the config if the permissions are too wide, but I can't reproduce it now.

If it's not needed, please ignore my suggestion. 100% agreed the procedure is already too many steps.

Reply:
Okay, thanks everyone. Let's not add it then. (For the record, I did try searching for anything related to git complaining about .gitignore being readable for everyone, but didn't find anything that should prevent us from sticking to what the step currently looks like.)